BZ-24655: fix : arrow-icon-positioning-of-select-component

adarshjuspay opened the pull request:
- remove arrow icon positioning for visual consistency
- remove duplicate CSS for cleaner, maintainable code

- Dev proof:
       Before:  
                    ![b1](https://github.com/juspay/svelte-ui-components/assets/167853238/25a998a1-d476-4fbc-af55-1075b577af91)
        After: 
                  ![a](https://github.com/juspay/svelte-ui-components/assets/167853238/82dbec27-c841-47d2-9ddd-425a75065e1f)

sinha-sahil commented on src/lib/Select/Select.svelte:
```diff
@@ -364,20 +362,6 @@
     --button-width: var(--multipleSelect-btn-width, 100%);
   }
 
-  .icon-container {
```
why did we remove this code ?  
Container is added intentionally when we want to keep the images small & increase click-ability. Lets avoid removing intentionally added code.

adarshjuspay replied:
its duplicate,
https://github.com/juspay/svelte-ui-components/pull/59/files#diff-db3ce7fe7922fbe616baccccb6e7ec305b3b35a23300e04e2365897b9541b557L395

adarshjuspay committed:
BZ-24655: fix : arrow icon positioning in select component
 - remove arrow icon positioning for visual consistency
 - remove duplicate CSS for cleaner, maintainable code
